packages/burner-connector/src/burnerConnector/burner.ts

```diff
@@ -130,6 +141,143 @@ export const burner = ({ useSessionStorage = false, rpcUrls = {} }: BurnerConfig
           return;
         }
 
+        if (method === "wallet_getCapabilities") {
+          // Accept params as [address, chainIds?]
+          const chainIdsRaw = (params as [string, string[]?])[1] as string[] | undefined;
+          // Get supported chains from config
+          const supportedChains = config.chains.map((chain) => numberToHex(chain.id));
+          // If no chainIds provided, use all supported
+          const chainIds = chainIdsRaw && chainIdsRaw.length > 0 ? chainIdsRaw : supportedChains;
+          const capabilities: Record<string, any> = {};
+          for (const chainId of chainIds) {
+            capabilities[chainId] = {
+              atomic: { status: "unsupported" },
+            };
+          }
+          return capabilities;
+        }
+
+        if (method === "wallet_sendCalls") {
+          // Define the type for params
+          type WalletSendCallsParams = {
+            version: string;
+            chainId: string;
+            from: string;
+            calls: Array<{
+              to: string;
+              data: string;
+              value: string;
+            }>;
+            atomicRequired: boolean;
+          };
+          const sendCallsParams = (params as [WalletSendCallsParams])[0];
+
+          if (sendCallsParams.atomicRequired) {
+            throw new Error("Atomic execution not supported");
+          }
+
+          // Execute calls sequentially for now
+          const requests = [];
+          let nonceIncrement = 0;
+          for (const call of sendCallsParams.calls) {
+            const request = await client.prepareTransactionRequest({
+              account: burnerAccount,
+              data: call.data as `0x${string}`,
+              to: call.to as `0x${string}`,
+              value: call.value ? hexToBigInt(call.value as `0x${string}`) : undefined,
+            });
+            requests.push({
+              ...request,
+              gas: (request.gas * GAS_MULTIPLIER) / 100n,
+              nonce: request.nonce + nonceIncrement,
+            });
+            nonceIncrement++;
+          }
+
+          const hashes = await Promise.all(requests.map((request) => client.sendTransaction(request)));
+
+          // Create a robust ID by concatenating transaction hashes, chain ID, and magic identifier
+          const id = concat([
+            ...hashes,
+            numberToHex(Number(sendCallsParams.chainId), { size: 32 }),
+            BURNER_MAGIC_IDENTIFIER,
+          ]);
+
+          return {
+            id,
+            capabilities: {
+              atomic: false,
+            },
```

Comment:
NIT: Should we follow this format here: 

```ts
capabilities: {
  "atomic": {
      "status": "supported"
   }
}
```

Not a big deal, since in `wallet_SendCalls` they have just mentioned `capabilities?: Record<string, any>;` 

But wherever they mentioned atomic inside `capabilities` they have mentioned this as type. 

Eg here: 

![Screenshot 2025-05-26 at 5 00 49 PM](https://github.com/user-attachments/assets/334fdf32-1ea2-4dfb-9094-ec8af5a3585d)

under `wallet_getCapabilities ` for `atomic` part

Anyways not a big deal though, just curious and we can leave it as it 🙌